src/sentry/event_manager.py

```diff
@@ -1797,10 +1798,16 @@ def _handle_regression(group: Group, event: Event, release: Optional[Release]) -
         transition_type="automatic",
         sender="handle_regression",
     )
-
     group.active_at = date
     group.status = GroupStatus.UNRESOLVED
     group.substatus = GroupSubStatus.REGRESSED
+    # intentionally fire the signal to update the cached Group
+    post_save.send(
+        sender=Group,
+        instance=group,
+        created=False,
+        update_fields=["last_seen", "active_at", "status", "substatus"],
+    )
```

Comment:
It could make sense here to only send the signal when `is_regression` is true. That means the update actually went through - otherwise we didn't actually update the group